New node running instructions with NaaS options (#11)
streamr-dev/documentation: docs/node-runners/run-a-node.md

**Using the config wizard**

_"Generate or import Ethereum private key"_

> You can generate a new private key or use one you already have. You can avoid having the private key of the wallet with your soon-to-be staked DATA stored in a plain text file by generating a new private key in this step and adding your staking wallet's public key as a Beneficiary Address once you are done configuring the node via the config wizard (highly recommended).

Reviewer: ```suggestion
You can generate a new private key or use one you already have. You can avoid having the private key of the wallet with your soon-to-be staked DATA stored in a plain text file by generating a new private key in this step and adding your staking wallet's public key as a *beneficiary address* once you are done configuring the node via the config wizard (highly recommended).
```

### Step 4: Add a Beneficiary Address to your node (optional)

> A Beneficiary Address allows you to only add the public key of the wallet with your staked DATA tokens to the config file instead of the private key. This way nothing sensitive is included in the config file, meaning if your device or server is comprised sometime in the future, your DATA tokens are safe. We highly recommend you use a Beneficiary Address. IF you

Reviewer: ```suggestion
A beneficiary address allows you to only add the public key of the wallet with your staked DATA tokens to the config file instead of the private key. By using a beneficiary address, if your node is comprised, the staked DATA tokens will not be at risk. We highly recommend you use a beneficiary address. IF you
```

Author: I'll make a separate commit to also fix a typo (comprised instead compromised).

> Be careful when you edit the JSON file. If you accidentally remove a character such as a curly bracket, the JSON format will be invalid and your node will fail to run.

Reviewer: ```suggestion
Exercise caution when you editing the config file. If you accidentally remove a character such as a curly bracket or use the wrong type of quotation symbol, the config file's JSON format will be invalid and your node will fail to run.
```

Author: Including this in the separate commit to fix "when you editing" = "when you edit"

**Windows PowerShell**

> Edit the config file with Notepad. This assumes that you've created the node folder in your Windows user's home folder. If that's not the case, then you need to correct the path. Replace `user` with your username for Windows.

Reviewer: ```suggestion
Edit the config file with Notepad. This assumes that you've created the node folder in your Windows user's home folder. If that's not the case, then you need to correct the path. Replace `user` with your logged in username.
```
